### DISCUSSION TITLE
Is horizontal scroll when scrolling element into view implemented?

### DISCUSSION THREAD

**Discussion by carriquiry-g** [hakimio/ngx-ui-tour#178] [Q&A] [answered]
I have a big table in my UI and some elements are far off the right side of the viewport.
In my tour steps I have configured a `scrollContainer` (this is correctly setup as some steps do scroll vertically) and explicitly set `centerAnchorOnScroll: true`. Anyways, I am not seeing any horizontal scrolling. I have gone through `scrolling.service.ts` and believe it should scroll horizontally.
I am not asking for a solution, just want to know if this feature is implemented. If not maybe it could be a feature request (in that case I will do it in the correct place).

**Comment by hakimio** [comment]
Does it scroll correctly when you call from the JS console:
```ts
            myAnchorEl.scrollIntoView({
                block: 'center',
                inline: 'center',
                behavior: 'smooth'
            });
```

**Reply by hakimio** [reply, marked as answer]
You can find the scrolling implementation in the following file:
https://github.com/hakimio/ngx-ui-tour/blob/26b33eda65e3328a155618779748ac689565ee8f/projects/ngx-ui-tour-core/src/lib/scrolling.service.ts#L35

**Reply by carriquiry-g** [reply]
Thanks for your answer. I guess my problem has to do with my HTML and not the library itself. I'll take a deeper look.
